campaigns.json

```diff
@@ -175,10 +175,12 @@
     "secondaries": {
       "/Hydjet_Quenched_Cymbal5Ev8_PbPbMinBias_5020GeV/HiFall15-75X_mcRun2_HeavyIon_v14_75X_mcRun2_HeavyIon_v14-v1/GEN-SIM": {
         "SiteWhitelist": [
-          "T2_US_MIT"
+          "T2_ES_CIEMAT",
+          "T1_RU_JINR_Disk"
         ]
       }
-    }
+    },
+  "secondary_AAA": false
```

Comment:
Are these hydjet datasets classical mixing and that's why we do not enable secondary AAA?

Reply:
Sure we seem to place the GenSim in the can not read remotely category

Reply:
@z4027163 what do you think about this?

Reply:
I think the issue with this https://cmsweb.cern.ch/reqmgr2/fetch?rid=pdmvserv_task_PPD-HINPbPbWinter16wmLHEGSHIMix-00002__v1_T_210526_132545_7657 might be that I forgot to add primary_AAA=false so I have added that. Even if there is a different issue because it is GenSim we have decided that it can not be read remotely.

Reply:
https://github.com/CMSCompOps/WmAgentScripts/issues/843